.eslintignore

```diff
@@ -3,8 +3,8 @@ node_modules
 /polaris-react/build
 /polaris-react/build-internal
 /polaris.shopify.com/.next
+/stylelint-polaris/data
```

Comment:
Content in this folder is dynamically generated, so shouldn't be linted

Reply:
Can we add that as a comment: e.g. `# Ignored as the contents of /data is generated`

Reply:
It's the same for all the other build and metadata folders above, so I don't really think it's worth it